BDLaethb6 and BDLaethb5

thea-m opened the pull request:
Dear all, 

Here is the long-awaited PR with the two Synaxaria.

A few points on both mss:
- I have updated the names to the best of my capabilities, but expect that there are still inconsistencies. I have worked on this over too long a period, it needs fresh eyes.

On BDLaethb6:
- a few pages were skipped during photographing, they are commented out in the description. I included them in our document for tasks at the library.
- in a4, I could not decide whether to create a record for "ወለተ፡ ቤርሳቤህ". I'm not sure whether this is a real name and refers to an actual person. Any opinions?
- F. 1 contains text identical to the one on f. 126 (not well visible, because f. 125v-126r were not photographed well), written by the main hand. The interpretation of this single page is not clear, maybe Eliana has also ideas about this? For now, it is not described as an additional codicological unit, but of course this might be changed.
- JJ writes in his article on the "Synkessar forger": “there is not a shred of evidence to connect [this ms] with King Fasiledes”: I disagree, as I believe his name was one originally included in many supplications. Therefore, I date this manuscript to his reign. The palaeography supports this, in my opinion. 
- the forgery history which I noted briefly in "history" might be expanded later with Jacopo's insights on the decorations.

I am fully aware that the revision will be a task similar in magnitude to the description, and will fall mainly to Massimo. Please take your time. As always, please read the files as displayed under "Files changed" so as not to review an old version, and please feel free to correct obvious mistakes and typos directly in the files.

Thanks everyone for your patience!

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethb5.xml:
```diff
@@ -2920,14 +2930,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <title type="complete"
                                             ref="LIT4035SenkessarDL#Teqe24SalMary">Salām-hymn to
-                                            Mary</title>
+                                            Māryām</title>
```
So, apparently she is not St Mary, right?

thea-m commented:
Dear Massimo, 
Thank you! Exactly, this was the principle we agreed on and that I worked with. Yes, so in this case for example, Gorgoryos is an Egyptian monk, so that's certainly not his original name, but because he is not known (to me), the Ethiopic version stayed. 
Doing a bit of it at a time seems very reasonable to me. It would probably be best if you then publish all your comments once you are finished with a daily session, so as not to risk loosing the review work. This will mean more e-mails (also for our colleagues here), but I think we shouldn't run the risk.
Further technical question: I also received a notification from you via email with a comment on ms_i1.3.25.5 in ath. b. 5 (on Māryām), but don't see this comment here on GitHub. Did you later delete it? If so, great, otherwise we need to check what's going on.
Thanks again!

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethb6.xml:
```diff
@@ -40,9 +43,8619 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    </msItem>
+                                    
+                                    <msItem xml:id="ms_i2.2.3.6">
+                                        <locus target="#"/>
```
`#4vc`?

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethb6.xml:
```diff
@@ -40,9 +43,8619 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <msItem xml:id="ms_i2.2.18.15">
+                                        <locus from="20vb" to="21vc"/>
+                                        <title type="complete" ref="LIT4035SenkessarDL#Mask18Yaqob"
+                                            >Commemoration of mār Yāʿqob the Egyptian</title>
```
What about either Mār (capitalized) or within `foreign xml:lang`?

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethb6.xml:
```diff
@@ -40,9 +43,8619 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        <title type="incomplete"
+                                            ref="LIT4035SenkessarDL#Mask24Gorgoryos">Commemoration
+                                            of Gorgoryos</title>
+                                        <note>The part of the text covered by the later added miniature is missing.</note>
```
Something in the wording sounds unnatural to me. @gnizla what do you think?
Perhaps "Part of the text is missing because covered by a miniature added at a later time" ?

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethb6.xml:
```diff
@@ -40,9 +43,8619 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        <locus target="26rb"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Mask24SalGorgoryos">Salām-hymn
+                                            to Gorgoryos and Quadratus</title>
```
Perhaps "St Quadratus", like in the preceding entry?

MassimoVilla reviewed (changes requested):
So, here comes the first set of comments. I reviewed all MS BDLaethb5 and the first month of MS BDLaethb6. Thank you Thea for all of this!

thea-m commented on OxfordBodleian/Juel-Jensen/BDLaethb5.xml:
```diff
@@ -2920,14 +2930,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <title type="complete"
                                             ref="LIT4035SenkessarDL#Teqe24SalMary">Salām-hymn to
-                                            Mary</title>
+                                            Māryām</title>
```
No, Māryām taḫārāmit (only the name is mentioned in the commemoration, nothing else)

MassimoVilla reviewed (changes requested):
Thank you again Thea. I think the next request will be the last one!

thea-m commented on OxfordBodleian/Juel-Jensen/BDLaethb6.xml:
```diff
@@ -40,9 +43,8619 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <msItem xml:id="ms_i2.4.22.7">
+                                        <locus target="#78ra"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Heda21Esmn
```
No, the IDs are only significant for the computers, they will not be printed anywhere. (FYI, the origin of this difference is that in the only ms of the "first recensions" in which this commemoration is attested, he is actually called Libānos)

thea-m replied:
So, what I mean is, yes, it is ok :)

MassimoVilla replied:
Ok, I was just wondering the reason of this difference, which is more than a mere spelling variance, as Liqānos and Libānos are different saints. Thank you for the explanation!

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethb6.xml:
```diff
@@ -204,46 +8825,424 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <desc type="Unclear">Colophon-type note written written by <ref target="#h2"/> in the lower margin, stating that the manuscript was written in the year of Grace 1744
+                                        at the time of <persName ref="PRS5617IyasuII"/>, and indicating the name of the scribe as <persName ref="PRS13549SardaD"/>.</desc>
+                                    <q xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩ አምላክ፡ አሜን።</hi>
+                                        ተጽሕፈ፡ ዝንቱ፡ መጽሐፈ፡ ስንክሣር<hi rend="rubric" rendition="#partialRubric">፥</hi> ዜና፡ ልደቶሙ፡ ወተዝካረ፡ እፈረቶሙ፡ ለአበዊነ፡ ቅዱሳን፡ 
```
I would mark up እፈረቶሙ፡ with `sic` or I would correct it:  the expected reading is እረፍቶሙ፡ ("their rest", i.e. death), and an exchange of letters has occurred here.

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethb6.xml:
```diff
@@ -204,46 +8825,424 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        ዓመተ፡ ምሕረት፡</date> በመዋዕለ<hi rend="rubric" rendition="#partialRubric">፥</hi> መንግሥቱ፡
+                                        ለንጉሥነ፥ <persName ref="PRS5617IyasuII"><add place="overstrike">ኢያሱ፡</add> <hi rend="rubric" rendition="#partialRubric">፪፨</hi></persName> <cb n="b"/> ወብእሲቱኒ፡ እግዝትነ፡ 
+                                        <!--<persName ref="">--><add place="overstrike">ወለተ፡ ቤርሳቤህ፡</add><!--</persName>--> ወፀሐፊሁ፡ <persName ref="PRS13549SardaD"><roleName type="title">መምሕር፡</roleName>
+                                        <hi rend="rubric">ሠርፀ፡ ድንግል።</hi></persName> ወእግዚአብሔር፡ ብሁተ፡ ሥልጣን፡ ያኑህ<hi rend="rubric" rendition="#partialRubric">፤</hi> ዘዘመነ፡ መንግሥቱ<hi rend="rubric" rendition="#partialRubric">፤</hi> ከመ፡ ኑሃ፡ ዘመኑ<hi rend="rubric" rendition="#partialRubric">፤</hi> ለኖህ፡ ወልደ፡ ማቱሳላ፡ ወያብርህ፡ አእይተ፡ አልባቢሁ<hi rend="rubric" rendition="#partialRubric">፤</hi> ወያግርር<hi rend="rubric" ren ...
```
I think አእይተ፡ stands for አእይንተ፡ "eyes (of their hearts)".

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethb6.xml:
```diff
@@ -204,46 +8825,424 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    በ፲፭ቱ፻፶፪፡ ነግሠ፡ ሚናስ።
+                                    በ፲ቱ፭፻፶፮ <gap reason="illegible"/> ሠርፀ፡ ድንግል።
+                                    በ፲ቱ፭፻፺፯ ነገሠ፡ ሱስንዮስ፡
+                                    በ፲ቱ፻ወ፮ተ፻፵፬ቱ፡ ነግሠ፡ ፋሲለደስ፡ እግዚእነ፡ ወአምላክነ፡ ረሰየነ፡ ድልዋነ፡ ከመ፡ ተአአምር፡ በበይናቲነ፡ በአምሃ፡ ቅድስት፡ ወን
```
I think ተአአምር፡ is ungrammatical. Perhaps አ is to be marked up with `<surplus/>`

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethb6.xml:
```diff
@@ -280,11 +9284,300 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                        <hi rend="rubric">ወበዛቲ፡ ዕለት፡ ካዕበ፡ ተዝካሮሙ፡</hi> ለፊሊቡና፡ ወከይና<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                        ወአስተርእዮተ፡ ርእሱ፡
+                        ለዮሐንስ፡ መጥም<add place="margin">ቅ፡ ወመርቴስ፡ ወማርሳ።</add>
+                        <seg type="supplication"><add place="margin">እግ<pb n="59r"/><cb n="a"/>አብሔር፡ ይምሐሮ፡ በጸሎቶሙ፡ ለፍቁሮሙ፡ 
```
ዚ in the word እግዚአብሔር፡ is apparently missing in the MS

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethb6.xml:
```diff
@@ -280,11 +9284,300 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                <div type="textpart" corresp="#ms_i2.5.32" xml:lang="gez">
+                    <!--subscription Tāḫśāś-->
+                    <ab>
+                        <pb n="#125r"/><cb n="b"/><lb n="24"/>
```
is # needed after `<pb n="`?

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethb6.xml:
```diff
@@ -280,11 +9284,300 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                    <!-\-introduction Ṭǝrr-\->
+                    to be checked from manuscript
+                    <ab>
+                    <pb n=""/><cb n=""/><lb n=""/>
```
`n=""` missing. Is the picture missing?

MassimoVilla reviewed (changes requested):
This is my last PR. Thank you once more Thea for this huge task!
As you have seen, I took the liberty to edit some very minor issues.

ElianaDalSasso commented on OxfordBodleian/Juel-Jensen/BDLaethb6.xml:
```diff
@@ -40,9 +43,8619 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                            <title type="incomplete" ref="LIT4035SenkessarDL#Terr"/>
+                            <note><locus target="#1"/> contains the beginning of 1st Ṭǝrr, written by the same hand as the rest of the manuscript,
+                                which is also contained in this manuscript on <locus target="#126"/>. It is unclear
+                            whether the leaf has been displaced or belonged originally to another manuscript.</note>
```
It is a rather puzzling question!
In my opinion, f.1 belonged to another manuscript since along the inner margin are sets of holes that do not correspond to the present sewing pattern. However, looking closely, it is possible to note further holes (partially covered by the parchment guard) that do match the present pattern.
I think that the f. 1 belonged to another manuscript (possibly slightly smaller) and for some reason it was selected to become part of the present one. The lower portion of blank parchment was added to repair the leaf and adjust the format, then the leaf was sewn to this manuscript. Later it must have become detached, the parchment guard has been added and the leaf has been linked to the sewing, in the present position.
It is just an hypothesis but it can work. However, the mending and the "links" around the sewing stations look modern to me.

ElianaDalSasso reviewed (commented):
Thank you Thea again for this massive work! I´ve only added a comment on f. 1 (let me know what you think) and about the condition of the ms.

thea-m commented on OxfordBodleian/Juel-Jensen/BDLaethb6.xml:
```diff
@@ -187,6 +8804,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         </item>
                                     </list>
                                 </collation>
+                                <condition key="good">The binding is slightly damaged, with the leather cover missing in the lower corners of the front board and all corners of the back board.
```
I'm sorry, I don't understand what you mean exactly. The back board is there, but the leather is damaged. Do you mean the leather cover is missing on the spine?

thea-m commented:
Hi everyone, 
I'm sorry that this took so long! I'm quite convinced by Eliana's argument, and have changed the description accordingly, splitting the record into two msParts. This entailed then a few modifications: since the additions by hand 2 and the decorations were added to both msParts presumably after they were joined, I have moved their descriptions to the first <physDesc> that pertains to the entire manuscript. I also moved there the description of the hands, as they are identical in p1 and p2. We will probably need to look at this together at a later point when working on the output to make sure that the description is understandable (Eliana's description of the binding and condition might also contribute to that). 
Please let me know if you agree with these decisions or if we need to reconsider. Thanks a lot!

MassimoVilla commented:
> Hi everyone, I'm sorry that this took so long! I'm quite convinced by Eliana's argument, and have changed the description accordingly, splitting the record into two msParts. This entailed then a few modifications: since the additions by hand 2 and the decorations were added to both msParts presumably after they were joined, I have moved their descriptions to the first that pertains to the entire manuscript. I also moved there the description of the hands, as they are identical in p1 and p2. We will probably need to look at this together at a later point when working on the output to make sure that the description is understandable (Eliana's description of the binding and condition might also contribute to that). Please let me know if you agree with these decisions or if we need to reconsider. Thanks a lot!

Thank you Thea! If the hand is the same in f. 1 and in the rest of the MS (as you also pointed out at the very beginning of this discussion), then the handDesc should fall in the general part, as you correctly did. This also necessarily implicates that the additiones were added by hand 2 at a later stage, so your opinion seems right to me.
The same question about the readability and intelligibility of the generated output might potentially occur with all composite mss, so we should discuss and verify a couple of example from composite mss and MS b.6 might be one.

ElianaDalSasso commented:
Thank you Thea! All fine! I think that also the description of the decoration might help!

thea-m commented:
I'll also merge this and note in the table that it's not yet reviewed by Jacopo.
